cfme/tests/automate/test_instance.py

```diff
@@ -354,3 +358,130 @@ def test_quota_source_value(request, entity, search, copy_quota_instance, generi
     provision_request.wait_for_request(method='ui')
     request.addfinalizer(lambda: provision_request.remove_request(method="rest"))
     assert result.validate(wait="60s")
+
+
+@pytest.fixture(scope='module')
+def setup(domain, klass, namespace):
+    """This fixture creates common domain, namespace, two classes, instance and method. This setup
+    is common for both parameterized tests"""
+
+    klass.schema.add_fields(*[{'name': state, 'type': 'State'}
+                              for state in [STATE1, STATE2, STATE3]])
+
+    # Adding method to execute via parent instance(state machine instance)
+    method = klass.methods.create(
+        name=fauxfactory.gen_alphanumeric(),
+        display_name=fauxfactory.gen_alphanumeric(),
+        location='inline',
+        script="""
+               \n$evm.log(:info, "Hello from method of parent instance")
```

Comment:
Are these newline characters `\n` necessary?

Reply:
yes. It is required to keep ruby code in well format while putting it in inline method data.